lib/ansible/modules/remote_management/oneview/oneview_logical_interconnect_group.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
```

Comment:
So the Copyright line has a specific format:
```python
# Copyright: (c) 2016-2017, Hewlett Packard Enterprise Development LP
```
Maybe also add a blank line before this one.